src/components/dialogs/custom-mui-dialog.tsx

```diff
@@ -9,7 +10,21 @@ import Button from '@mui/material/Button';
 import makeStyles from '@mui/styles/makeStyles';
 import { SubmitButton } from '@gridsuite/commons-ui';
 
-const useStyles = makeStyles((theme) => ({
+interface ICustomMuiDialog {
+    open: boolean;
+    formSchema: any;
+    formMethods: any;
+    onClose: (event: React.MouseEvent) => void;
+    onSave: (data: any) => void;
+    onValidationError?: (errors: any) => void;
```

Comment:
it could be nice to find the proper ways to type "formSchema", "formMethods", "onSave" and "onValidationError" properties